tests/integration/paste.test.js

```diff
@@ -0,0 +1,78 @@
+import request from 'supertest-as-promised';
+import server from '../../server';
+import chai from 'chai';
+import moduleLoader from 'es6-module-loader';
+
+var System = moduleLoader.System;
+
+var app;
+chai.should();
+
+System.import('chai-datetime')
+    .then((dateTime) => chai.use(dateTime));
+
+describe.skip('INTEGRATION-TESTS::Paste API tests', () => {
+    beforeEach(() => app = server.start());
+
+    afterEach((done) => app.close(done));
+
+    it('should return 201 and a slug when a post is successful', () => {
+        return request(server)
+            .post('api/paste', {
+                body: 'test data'
+            })
+            .expect(201)
+            .expect(function(res) {
+                res.body.should.have.property('slug').and.exist();
+                res.body.should.have.property('createdById').and.exist();
+            });
+    });
+
+    it('should return 500 when post fails', () => {
+        true.should.be.false;
```

Comment:
all these tests are commented out, but where you see that its because they obviously aren't implemented. I usually put in some forced assert fail so that they stand out during development.